nethserver-samba.spec

```diff
@@ -12,7 +12,8 @@ Obsoletes: nethserver-ibays
 Requires: samba
 Requires: tdb-tools
 Requires: nethserver-ibays
-Requires: sssd-libwbclient
+Obsoletes: sssd-libwbclient
```

Comment:
I'd remove the package in an action to avoid YUM issues: the nethserver-samba RPM actually doesn't _obsolete_ anything.

Reply:
You can't access the RPM database inside the update event.
Already tried, already failed.

Reply:
Can we fix it with the "alternative" command? IIRC it is a just a symlink to a shared library